.github/docker_ubuntu_bionic/DockerFile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update -y && apt-get -y install --no-install-recommends \
     libpng16-16 \
     # install libfontconfig for qt
     libfontconfig && \
+    libicu && \
```

Comment:
Do you test only TUI?, or also GUI?

Reply:
I do test both, although my GUI is not yet in a usable state (developing the GUI takes me more time than the backend :sweat_smile:)

But I did manage to test all the GUI related things I wrote so far, see e.g. [here](https://github.com/philbucher/KratosSalomePlugin/blob/master/tests/test_plugin_controller.py)

the final goal is for me to have only a small GUI and to use [LiteKratos](https://github.com/KratosMultiphysics/LiteKratos) for the problemtype logic